recipes-extended/xen/files/libxl-fix-flr.patch

```diff
@@ -0,0 +1,154 @@
+Index: xen-4.6.6/tools/libxl/libxl.c
+===================================================================
+--- xen-4.6.6.orig/tools/libxl/libxl.c
++++ xen-4.6.6/tools/libxl/libxl.c
+@@ -1604,7 +1604,13 @@ static void domain_destroy_callback(libx
+         LOG(ERROR, "unable to destroy guest with domid %u", dis->domid);
+         dds->rc = rc;
+     }
+-
++    if(dis->pciw) {
++        for (int i = 0; i < dis->pciw->num_devs; i++) {
++            libxl__device_pci_reset(gc, dis->pciw->pcidevs[i].domain, dis->pciw->pcidevs[i].bus, dis->pciw->pcidevs[i].dev, dis->pciw->pcidevs[i].func);
```

Comment:
Just a stylistic comment, but you could have done a local reference variable here(see below) and it would have reduced keystrokes and line length
```c
        for (int i = 0; i < dis->pciw->num_devs; i++) {
             libxl_device_pci *pcidev = &dis->pciw->pcidevs[i];
￼             libxl__device_pci_reset(gc, pcidev->domain, pcidev->bus, pcidev->dev, pcidev->.func);
        }
```

Reply:
Yea, I was aiming for fewer total lines, but if the shorter line length is desired I can edit the patch